fnet/src/vespa/fnet/frt/rpcrequest.cpp

```diff
@@ -129,7 +129,7 @@ void
 FRT_RPCRequest::SubRef()
 {
     assert(_refcnt > 0);
-    if (vespalib::Atomic::postDec(&_refcnt) == 1) {
+    if (_refcnt.fetch_sub(1) == 1) {
```

Comment:
Both the implicit atomic load in the `assert` and the `fetch_sub` will trigger full fences. If this is a problem, can assert on the returned value from `fetch_sub` instead to reduce fence count by 1.